init.lua

```diff
@@ -1,3 +1,4 @@
+print "hi"
```

Comment:
can you remove this line plz

Reply:
I told myself to remove that before comitting 😞